prac_05/wimbledon.py

```diff
@@ -0,0 +1,27 @@
+
+
+infile = open("wimbledon.csv", "r", encoding="utf-8-sig")
+CHAMPION_INDEX = 2
+NATIONALITY_INDEX = 1
+champions = []
+nationalities = []
+win_count = {}
```

Comment:
Name 'win_count' is hard to understand that it is dictionary of champions and their win counts